rbi/core/gc.rbi

```diff
@@ -50,6 +50,26 @@ module GC
   # Returns information about the most recent garbage collection.
   def self.latest_gc_info(*_); end
 
+  #  Returns whether or not automatic compaction has been enabled.
```

Comment:
Was this documentation generated with [`syn-rdoc`](https://github.com/sorbet/sorbet/blob/master/rbi/tools/sync-rdoc.rb) or written manually?

```suggestion
  # Returns whether or not automatic compaction has been enabled.
```

Reply:
I copy / pasted this from `gc.rb`, so this could be my copy paste mistake